bounter/count_min_sketch.py

```diff
@@ -128,6 +128,9 @@ def table_size(width, depth=4, log_counting=None):
     def __getitem__(self, key):
         return self.cms.get(key)
 
+    def __contains__(self, item):
+        return self.cms.get(item)
```

Comment:
Does this return a boolean?

Reply:
yes

It runs like described in the comment above: 
https://github.com/RaRe-Technologies/bounter/pull/38#issue-225933667